share/models/creative.py

```diff
@@ -12,18 +12,22 @@
 class AbstractCreativeWork(ShareObject, metaclass=TypedShareObjectMeta):
     title = models.TextField(blank=True)
     description = models.TextField(blank=True)
+    identifiers = ShareManyToManyField('Identifier', through='WorkIdentifier')
 
     # Used to determine if something should be surfaced in ES or not
     # this may need to be renamed later
     is_deleted = models.BooleanField(default=False)
 
+    # True for works that only exist because they were referenced by other works.
+    # Should be false if there's any information beyond an identifier.
+    # Don't put stubs in ES.
+    is_stub = models.BooleanField(default=False)
```

Comment:
That makes perfect sense, I'll remove it.

I mostly added `is_stub` not for ES, but to distinguish between an empty change that means "no changes, skip" and an empty change that means "create an empty object", but I can do that without an actual field.